<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registration</class>
 <widget class="QMainWindow" name="registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(75, 75, 75);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #00bcd4;                   /* Teal color to match your theme */
    font-size: 32px;                  /* Large font size for prominence */
    font-weight: bold;                /* Bold text */
    letter-spacing: 2px;              /* Add some spacing between letters */
    text-align: center;               /* Center the headline */
    padding: 10px;                    /* Padding around the text */
    background-color: transparent;    /* Transparent background */
    text-shadow: 2px 2px 5px #000;    /* Soft shadow for better readability */
    border-bottom: 3px solid #00bcd4; /* Adds a bottom border for emphasis */
    border-radius: 5px;               /* Rounded bottom border */
}
</string>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #f0f0f0;                
    font-size: 20px;                
    font-weight: bold;           
    letter-spacing: 1px;           
    padding: 5px;                   
    background-color: transparent;  
    text-shadow: 1px 1px 2px #000;  
}

QLabel#titleLabel {
    font-size: 28px;                
    font-weight: bold;              
    letter-spacing: 2px;          
    padding: 10px;               
    color: #00bcd4;            
    background-color: transparent; 
    border-bottom: 2px solid #00bcd4; 
    text-align: center;             
    text-shadow: 2px 2px 5px #000;  
    border-radius: 5px;            
}
</string>
    </property>
    <property name="text">
     <string>Full Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #f0f0f0;                
    font-size: 20px;                
    font-weight: bold;           
    letter-spacing: 1px;           
    padding: 5px;                   
    background-color: transparent;  
    text-shadow: 1px 1px 2px #000;  
}

QLabel#titleLabel {
    font-size: 28px;                
    font-weight: bold;              
    letter-spacing: 2px;          
    padding: 10px;               
    color: #00bcd4;            
    background-color: transparent; 
    border-bottom: 2px solid #00bcd4; 
    text-align: center;             
    text-shadow: 2px 2px 5px #000;  
    border-radius: 5px;            
}
</string>
    </property>
    <property name="text">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #f0f0f0;                
    font-size: 20px;                
    font-weight: bold;           
    letter-spacing: 1px;           
    padding: 5px;                   
    background-color: transparent;  
    text-shadow: 1px 1px 2px #000;  
}

QLabel#titleLabel {
    font-size: 28px;                
    font-weight: bold;              
    letter-spacing: 2px;          
    padding: 10px;               
    color: #00bcd4;            
    background-color: transparent; 
    border-bottom: 2px solid #00bcd4; 
    text-align: center;             
    text-shadow: 2px 2px 5px #000;  
    border-radius: 5px;            
}
</string>
    </property>
    <property name="text">
     <string>Phone number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #1e1e1e;      /* Dark gray background */
    color: white;                   /* White text */
    padding: 8px 12px;              /* Padding for more space inside */
    border: 2px solid #5a5a5a;      /* Subtle gray border */
    border-radius: 10px;            /* Rounded corners */
    font-size: 16px;                /* Adjust font size */
}

QLineEdit:focus {
    border: 2px solid #00bcd4;      /* Light blue border when focused */
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #1e1e1e;      
    color: white;                 
    padding: 8px 12px;              
    border: 2px solid #5a5a5a;    
    border-radius: 10px;         
    font-size: 16px;                
}

QLineEdit:focus {
    border: 2px solid #00bcd4;     
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #1e1e1e;      
    color: white;                 
    padding: 8px 12px;              
    border: 2px solid #5a5a5a;    
    border-radius: 10px;         
    font-size: 16px;                
}

QLineEdit:focus {
    border: 2px solid #00bcd4;     
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #f0f0f0;                
    font-size: 20px;                
    font-weight: bold;           
    letter-spacing: 1px;           
    padding: 5px;                   
    background-color: transparent;  
    text-shadow: 1px 1px 2px #000;  
}

QLabel#titleLabel {
    font-size: 28px;                
    font-weight: bold;              
    letter-spacing: 2px;          
    padding: 10px;               
    color: #00bcd4;            
    background-color: transparent; 
    border-bottom: 2px solid #00bcd4; 
    text-align: center;             
    text-shadow: 2px 2px 5px #000;  
    border-radius: 5px;            
}
</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #1e1e1e;      
    color: white;                 
    padding: 8px 12px;              
    border: 2px solid #5a5a5a;    
    border-radius: 10px;         
    font-size: 16px;                
}

QLineEdit:focus {
    border: 2px solid #00bcd4;     
}
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #f0f0f0;                
    font-size: 20px;                
    font-weight: bold;           
    letter-spacing: 1px;           
    padding: 5px;                   
    background-color: transparent;  
    text-shadow: 1px 1px 2px #000;  
}

QLabel#titleLabel {
    font-size: 28px;                
    font-weight: bold;              
    letter-spacing: 2px;          
    padding: 10px;               
    color: #00bcd4;            
    background-color: transparent; 
    border-bottom: 2px solid #00bcd4; 
    text-align: center;             
    text-shadow: 2px 2px 5px #000;  
    border-radius: 5px;            
}
</string>
    </property>
    <property name="text">
     <string>Conform Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #1e1e1e;      
    color: white;                 
    padding: 8px 12px;              
    border: 2px solid #5a5a5a;    
    border-radius: 10px;         
    font-size: 16px;                
}

QLineEdit:focus {
    border: 2px solid #00bcd4;     
}
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #00bcd4;     
    color: white;                 
    font-size: 18px;              
    font-weight: bold;            
    border-radius: 15px;            
    padding: 12px 20px;            
}

QPushButton:hover {
    background-color: #008c9e;      
}

QPushButton:pressed {
    background-color: #006e7d;    
}
</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #1e1e1e; 
    color: white;               
    border: 2px solid #5a5a5a;  
    border-radius: 10px;      
    padding: 10px 20px;       
    font-size: 15px;            
}




</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
